src/ape/api/providers.py

```diff
@@ -689,6 +688,7 @@ def send_transaction(self, txn: TransactionAPI) -> ReceiptAPI:
             else self.network.required_confirmations
         )
         receipt = self.get_transaction(txn_hash.hex(), required_confirmations=req_confs)
+        logger.info(f"Submitted {receipt.txn_hash} (gas_used={receipt.gas_used})")
```

Comment:
Note that `web3.eth.wait_for_transaction_receipt` is blocking, so that you will have to wait a decent number of confs to see `gas_used` from the receipt.

Probably should be seperated into two info log events:
1. f"Submitted {txn_hash.hex()} ({explorer_txn_url})
2. f"Confirmed {receipt.txn_hash} (gas_used={receipt.gas_used})

Number (2) might best live in `ReceiptAPI` come to think of it

Reply:
Oh that makes sense.